api/proto/core.proto

```
@@ -0,0 +1,93 @@
+}
+
+message NotifyImagePushedResponse {
+  bool accepted = 1;
+  string message = 2;
+}
+
+// RouteChangeEvent for streaming route changes
+message RouteChangeEvent {
+  enum ChangeType {
+    CHANGE_TYPE_UNSPECIFIED = 0;
+    INVALIDATE = 1;          // Specific domain invalidated
```

Buf (1.64.0)

[error] Enum value name "INVALIDATE" should be prefixed with "CHANGE_TYPE_".

(ENUM_VALUE_PREFIX)